core/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,334 @@
+// .su-main-nav--right          - right justify the nav
+//
+// Markup: ../templates/components/main-nav/main-nav.twig
+//
+// Style guide: Components.Main Navigation
+//
+.su-main-nav {
+  @include font-smoothing;
+  font-family: $font-sans;
+  position: relative;
+  z-index: 9999;
+
+  ul {
+    @include margin(0 null);
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    list-style: none;
+    background-color: $color-black;
+  }
+
+  //top level styles
+  > ul {
+    > li {
+      @include margin(null null 0);
+
+      > a {
+        font-weight: 600;
+      }
+    }
+
+    @include grid-media('lg') {
+      @include padding(null null null 0);
+      flex-direction: row;
+      flex-wrap: wrap;
+      background-color: transparent;
+
+      > li {
+        > a {
+          @include fancy-hover;
+          @include padding(null 0);
+          @include margin(0 1.16em 0 0);
+          font-size: 1.05em;
+          border-bottom: 0;
+
+          &:active {
+            font-weight: 600;
+          }
+
+          &:first-child {
+            > a {
+              @include margin(null null null 0);
+            }
+          }
+
+          &:last-child {
+            > a {
+              @include margin(null 0 null null);
+            }
+          }
+
+        }
+
+        &.su-main-nav__item--current {
+          > a {
+            font-weight: 600;
+          }
+        }
+      }
+    }
+  }
+
+  //general deeper levels
+  li {
+    @include margin(null null 0);
+
+    a {
+      @include fancy-hover($color: $color-white, $active: #ec0513, $position: 'left');
+      @include padding(.8em 2.4em .8em 1.2em);
+      display: block;
+      color: $color-white;
+      font-weight: 400;
+      font-size: .9em;
+      border-bottom: $color-cool-grey solid 1px;
+      outline: 0;
+
+      &:active {
+        font-weight: 700;
+      }
+    }
+
+    > ul {
+      display: none;
+    }
+
+    &.su-main-nav__item--expanded {
+      > ul {
+        display: flex;
+      }
+    }
+
+    &.su-main-nav__item--current {
+      > a {
+        font-weight: 700;
+
+        &::before {
+          visibility: visible;
+          transform: scaleY(1);
+          background-color: #ec0513;
+        }
+
+        &:hover,
+        &:focus {
+          &::before {
+            left: 6px;
+          }
+        }
+      }
+    }
+
+    &.su-main-nav__item--parent {
+      > a {
+        &::after {
+          background: url("#{$image-path}/plus-white.svg") no-repeat 0 0;
+          background-size: 100%;
+          content: "";
+          display: inline-block;
+          position: absolute;
+          right: 1em;
+          top: 1.03em;
+          height: .9em;
+          width: .9em;
+
+          @include grid-media('lg') {
+            @include margin(null null 1px 6px);
+            background: url("#{$image-path}/caret-down-black.svg") no-repeat 0 0;
+            background-size: 100%; //have to repeat here for some reason
```

Comment:
🤔 Code comments like this usually give me a raised eyebrow. This looks like a good opportunity to connect and have a live debugging session with someone.